FAQ Improvements (#616)
xmtp/xmtp-dot-org: docs/faq.md

:::

> ### Does XMTP works with smart contract wallets?

Reviewer: ```suggestion
### Does XMTP work with smart contract wallets?
```

We don't provide a direct method to fetch all XMTP-enabled wallet addresses in bulk. However, you could consider utilizing a third-party service like Airstack. They have compiled an extensive list of XMTP addresses by querying our canMessage API over time at a rate that avoids hitting the limits. This approach could be a time-efficient alternative for you.

> ## Group chats

Reviewer: ```suggestion
### Does XMTP support group chat?
```

Also may I suggest that we add this question to the Message patterns section? https://xmtp.org/docs/faq#message-patterns I think it would fit right in there. =)

> Despite XMTP efforts to build group chat on XMTP v2 (which all our SDKs currently support), the protocol limitations prevent us from achieving the required security level for group conversations. The good news is that we're shifting our focus to XMTP v3, with the Alpha launch coming in the next couple of weeks. This opens doors for a secure group chat beginning of next year, aligning with our commitment to maintaining robust security standards. More details to share soon, so stay tuned!

Reviewer: How about something like the following? =) In documentation, we aim to not make forward-looking statements about deliverables and dates. The closest we come is the Roadmap, which lives outside of the documentation. Also removed references to XMTP v3 and alpha because I don't think the notion of v3 exists anymore. I think releases are going to happen on a per-feature basis. =) Provided a link to the community forum where people can voice any questions or concerns about this statement.

```suggestion
Not yet. Despite efforts to build group chat with XMTP v2 (the current protocol version), protocol limitations prevented group chat from meeting XMTP's robust security standards. Efforts have shifted to updating the protocol to support secure group chat. More details to come. 

Have other questions or feedback about group chat? Post to the [XMTP Community Forums](https://community.xmtp.org/).
```
